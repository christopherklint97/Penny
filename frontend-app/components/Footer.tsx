export default function Footer() {
  return (
    <nav className="p-3 h-16 italic underline absolute bottom-0 w-full bg-blue-700 text-gray-50">
      <div className="md:w-5/6 flex flex-row justify-end mx-auto">
        <a
          href="https://github.com/christopherklint97/Penny"
          target="_blank"
          rel="noreferrer"
          className="hover:text-gray-300"
        >
          Made with love on Github
        </a>
      </div>
    </nav>
  )
}
