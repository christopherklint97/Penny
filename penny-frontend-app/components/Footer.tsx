import { Navbar, Nav, NavItem, NavLink } from 'reactstrap'
import styles from './Footer.module.css'

export function Footer() {
  return (
    <Navbar expand="md" className={styles.blueBg}>
      <Nav
        className={`${styles.whiteText} ml-auto d-flex justify-content-center`}
        navbar
      >
        <NavItem>
          <NavLink
            href="https://github.com/christopherklint97/Penny"
            className={`${styles.whiteText} yellowtail`}
          >
            Made with love on Github
          </NavLink>
        </NavItem>
      </Nav>
    </Navbar>
  )
}
