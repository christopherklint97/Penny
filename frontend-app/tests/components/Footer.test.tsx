import React from 'react'
import { render } from '../testUtils'
import Footer from '../../components/Footer'

describe('Footer component', () => {
  it('renders without crashing', function () {
    render(<Footer />)
  })

  it('matches snapshot', () => {
    const { asFragment } = render(<Footer />, {})
    expect(asFragment()).toMatchSnapshot()
  })
})
