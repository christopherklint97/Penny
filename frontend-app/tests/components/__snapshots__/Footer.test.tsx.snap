// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component matches snapshot 1`] = `
<DocumentFragment>
  <nav
    class="p-3 h-14 inset-x-0 italic underline absolute bottom-0 w-full bg-blue-300 text-gray-800"
  >
    <div
      class="md:w-5/6 flex flex-row justify-end mx-auto"
    >
      <a
        class="hover:text-gray-600"
        href="https://github.com/christopherklint97/Penny"
        rel="noreferrer"
        target="_blank"
      >
        Made with love on Github
      </a>
    </div>
  </nav>
</DocumentFragment>
`;
